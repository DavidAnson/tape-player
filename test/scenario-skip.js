// @ts-check

"use strict";

const tape = require("tape");
require("../index.js");

tape("first test", (test) => {
  test.plan(1);
  test.pass();
});

tape.skip("middle test", (test) => {
  test.plan(2);
  test.pass();
  test.pass();
});

tape("last test", (test) => {
  test.pass();
  test.end();
});
