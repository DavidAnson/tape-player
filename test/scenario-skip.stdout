first test
last test

Tests:    2
Asserts:  2
Failures: 0
